corehq/apps/domain/templates/login_and_password/login.html

```diff
@@ -1,10 +1,19 @@
 {% extends "style/base_page.html" %}
 {% load i18n %}
+{% load hq_shared_tags %}
+{% load compress %}
 
 {% block title %}{% trans "Log into CommCare HQ" %}{% endblock title %}
 
 {% block page_content %}
     {% block login-content %}
     {% include "login_and_password/partials/login_full.html" %}
     {% endblock %}
+
+    {% if implement_password_hashing %}
+        {% compress js %}
+            {{ block.super }}
+            <script src="{% static 'domain/js/encoder.js' %}"></script>
```

Comment:
re: https://github.com/dimagi/commcare-hq/pull/15490#discussion_r107991151

I'm not sure if this is correct yet. Looks like it's in the `page_content` block. I think it should be in the `js` block.